client/packages/system/src/Patient/CreatePatientModal/DefaultCreatePatientUISchema.json

```diff
@@ -29,12 +29,12 @@
           "label": "label.gender",
           "options": {
             "show": [
-              ["MALE", "Male"],
-              ["FEMALE", "Female"],
-              ["TRANSGENDER_MALE", "Transgender Male"],
-              ["TRANSGENDER_FEMALE", "Transgender Female"],
-              ["NON_BINARY", "Non-binary"],
-              ["UNKNOWN", "Unknown"]
+                  ["MALE", "gender.male"],
```

Comment:
I saw other reports using some kinda tag like `T#gender.female`. What's with that and why not here?

Reply:
Cause those are being served from backend, backend translates.. this one isn't saved in DB >.< so have to handle it somehow
https://github.com/msupply-foundation/open-msupply/pull/6953#discussion_r1999975776 <-